### PR TITLE
cardano ledgerjs package update

### PR DESCRIPTION
**Pull request by pavanvora**
### 📝 Description

@cardano-foundation/ledgerjs-hw-app-cardano package update to support Cardano App v6.0.1

### ❓ Context

- **Impacted projects**: `llc` 
- **Linked resource(s)**: `` 

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -63390,3 +63373,7 @@ packages:
     dependencies:
       commander: 2.20.3
     dev: false
+
+settings:
+  autoInstallPeers: false
+  excludeLinksFromLockfile: false
```
Comment (haammar-ledger) [inline pnpm-lock.yaml:63379]: Not sure what this change does, but it doesn't seem required, is it ?
If not can you please remove it? (by re-generating this file)

**Review by haammar-ledger — CHANGES_REQUESTED** [review]
Please just add a changeset with `pnpm changeset`

File: pnpm-lock.yaml (hunk 1 above)
Comment (pavanvora) [reply pnpm-lock.yaml:63379]: it automatically got moved from top to bottom of the file and I tried regenerating and it is doing same. should I just revert those 3 line changes?

Comment (haammar-ledger) [reply pnpm-lock.yaml:63379]: Nevermind, I didn't see that it just got moved around in the file indeed, and besides this change seems to be on `develop` too already. All good :+1:

**Pull request merged**